src/payment/entities/payment.entity.ts

```diff
@@ -1 +1,34 @@
-export class Payment {}
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import * as mongoose from 'mongoose';
+import { Deposit } from '../../deposit/entities/deposit.entity';
+import { User } from '../../user/entities/user.entity';
+import { PaymentMethod } from '../enums/paymentMethod.enum';
+
+@Schema()
+export class Payment {
+  @Prop({
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
+    required: true,
+  })
+  user: User;
+
+  @Prop({
+    type: String,
+    required: true,
```

Comment:
wydaje mi się że ten test przechodzi, bo tutaj jeszcze trzeba mu powiedzieć, że to property korzysta z enum, `enum: PaymentMethod`